calico_node/Makefile

```diff
@@ -184,8 +183,8 @@ dist/startup: $(STARTUP_FILES) vendor
 		-v $(VERSIONS_FILE):/versions.yaml:ro \
         -e VERSIONS_FILE=/versions.yaml \
 	  	$(CALICO_BUILD) sh -c '\
-	    	cd /go/src/$(PACKAGE_NAME) && \
-	    	make startup'
+			cd /go/src/$(PACKAGE_NAME) && \
```

Comment:
I switched the line to be all tabs.  It is meant to be indented more than the `sh` line above